src/marqo/instance_mappings.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+
+from typing import Dict, Optional
+
+
+class InstanceMappings(ABC):
```

Comment:
We need to add a clear message here that this doesn't support cases where you have the same index name on more than 1 URL cluster